site/gatsby-site/package.json

```diff
@@ -57,6 +58,7 @@
     "graphql-http": "^1.18.0",
     "i18next": "^20.6.1",
     "is-absolute-url": "^3.0.3",
+    "lmdb": "^2.8.0",
```

Comment:
I don't see any reference to this package in the code. If this is not used please remove and re-generate the `package-lock.json` file.

Reply:
removed lmdb